README.md

```diff
@@ -1,6 +1,35 @@
 [[_TOC_]]
 
```

Comment:
Let's remove this as GitHub doesn't support table of contents embedded in readme
```suggestion
```